components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.js

```diff
@@ -151,12 +120,6 @@ document.observe('xwiki:dom:loading', function() {
                 item._restriction = null;
               }
             }
-            if (item._customOptions.queryProcessor && item._restriction) {
-              item._customOptions.queryProcessor = Object.clone(item._customOptions.queryProcessor);
-              item._customOptions.queryProcessor.restriction = {
-                'term_category' : item._restriction
```

Comment:
Regression: searching in only one subsection of HPO no longer works, results from the entire HPO are returned.